gyp/pylib/gyp/easy_xml.py

```diff
@@ -115,11 +116,10 @@ def WriteXmlIfChanged(content, path, encoding='utf-8', pretty=False,
   xml_string = XmlToString(content, encoding, pretty)
   if win32 and os.linesep != '\r\n':
     xml_string = xml_string.replace('\n', '\r\n')
-    
-  try:
-    xml_string = xml_string.encode(encoding)
-  except Exception:
-    xml_string = unicode(xml_string, 'latin-1').encode(encoding)
+  
+  default_encoding = locale.getdefaultlocale()[1]
+  if default_encoding != encoding.upper():
```

Comment:
Ok, this is my negligence, I mistakenly thought it  is capitalized, by this result:

![image](https://cloud.githubusercontent.com/assets/1730073/26276068/1759c6dc-3da2-11e7-9d94-a8606877dc7d.png)